lib/clean_sudoku.rb

```diff
@@ -0,0 +1,13 @@
+class CleanSudoku
+    def self.clean_board(board)
+        cleaned_board = board.split("\n").map do |row|
+            next if row == "------+------+------"
+            row = row.gsub("|", "").strip.split(" ").map do |cell|
+                cell.to_i
+            end
```

Comment:
You could use a shorthand version:
```suggestion
            row = row.gsub("|", "").strip.split(" ").map { |cell| cell.to_i }
```

Or even shorter:
```suggestion
            row = row.gsub("|", "").strip.split(" ").map(&:to_i)
```

https://www.brianstorti.com/understanding-ruby-idiom-map-with-symbol/